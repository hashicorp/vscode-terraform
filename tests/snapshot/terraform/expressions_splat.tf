[for o in var.list : o.id]

var.list[*].id

var.list[*].interfaces[0].name
