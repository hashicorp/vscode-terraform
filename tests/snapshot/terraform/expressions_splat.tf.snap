>[for o in var.list : o.id]
#^ scope.terraform punctuation.section.brackets.begin.terraform
# ^^^ scope.terraform keyword.control.terraform
#    ^ scope.terraform
#     ^ scope.terraform variable.other.readwrite.terraform
#      ^ scope.terraform
#       ^^ scope.terraform keyword.operator.word.terraform
#         ^ scope.terraform
#          ^^^ scope.terraform support.constant.terraform
#             ^ scope.terraform keyword.operator.accessor.terraform
#              ^^^^ scope.terraform variable.other.member.terraform
#                  ^ scope.terraform
#                   ^ scope.terraform keyword.operator.terraform
#                    ^ scope.terraform
#                     ^ scope.terraform variable.other.readwrite.terraform
#                      ^ scope.terraform keyword.operator.accessor.terraform
#                       ^^ scope.terraform variable.other.member.terraform
#                         ^ scope.terraform punctuation.section.brackets.end.terraform
>
>var.list[*].id
#^^^ scope.terraform support.constant.terraform
#   ^ scope.terraform keyword.operator.accessor.terraform
#    ^^^^ scope.terraform variable.other.member.terraform
#        ^^^ scope.terraform
#           ^ scope.terraform keyword.operator.accessor.terraform
#            ^^ scope.terraform variable.other.member.terraform
>
>var.list[*].interfaces[0].name
#^^^ scope.terraform support.constant.terraform
#   ^ scope.terraform keyword.operator.accessor.terraform
#    ^^^^ scope.terraform variable.other.member.terraform
#        ^^^ scope.terraform
#           ^ scope.terraform keyword.operator.accessor.terraform
#            ^^^^^^^^^^ scope.terraform variable.other.member.terraform
#                      ^^^ scope.terraform
#                         ^ scope.terraform keyword.operator.accessor.terraform
#                          ^^^^ scope.terraform variable.other.member.terraform
>